INSTALL.md

```diff
@@ -433,6 +433,21 @@ Calls to `offload_dgemm` also accept pointers on GPU or a combination of them.
 - See <https://brehm-research.de> for more information.
 -->
 
+### 2v. DeePMD-kit (optional, wider range of interaction potentials)
+
+DeePMD-kit - Deep Potential Molecular Dyanmics. Support for DeePMD-kit can be enabled via the flag
+`-D__DEEPMD`.
+
+- Please set `--deepmd-mode` to `gpu` if GPU support is needed, and load CUDA environment before
+  installing.
```

Comment:
This seems to be a reference to the toolchain. So I would rather drop it. Same with the other two points.